addons/web/static/tests/views/fields/many2many_tags_avatar_field_tests.js

```diff
@@ -76,7 +77,7 @@ QUnit.module("Fields", (hooks) => {
         assert.strictEqual(
             target.querySelector(".o_field_many2many_tags_avatar.o_field_widget .o_avatar img")
                 .dataset.src,
-            "/web/image/partner/2/avatar_128",
+            `${getOrigin()}/web/image/partner/2/avatar_128`,
```

Comment:
prefer full assert rather than includes (I want to see the unique key in all tests)